R/testdata.R

```diff
@@ -1083,10 +1129,10 @@ dataset_test_worker <-
 
         ## Check traits are not only NAs
```

Comment:
Maybe better `Check that not all trait names are NAs`. When I read header I thought this was about trait values until I looked at code.